libgraph/src/PropertyGraph.cpp

```diff
@@ -221,6 +221,20 @@ katana::PropertyGraph::Make(katana::GraphTopology&& topo_to_assign) {
       EntityTypeManager{}, EntityTypeManager{});
 }
 
+katana::Result<std::unique_ptr<katana::PropertyGraph>>
+katana::PropertyGraph::Make(
```

Comment:
So this got removed somehow?

Reply:
Yeah. In my head I had added an overload. In my computer I had overwritten the existing overload.